examples/CMakeLists.txt

```diff
@@ -1,12 +1,4 @@
-add_subdirectory(3pt_stencil)
-add_subdirectory(asynchronous_stopping_criterion)
-add_subdirectory(custom_matrix_format)
-add_subdirectory(ginkgo_overhead)
-add_subdirectory(inverse_iteration)
-add_subdirectory(minimal_solver_cuda)
-add_subdirectory(papi_logging)
-add_subdirectory(preconditioned_solver)
-add_subdirectory(poisson_solver)
-add_subdirectory(ranges)
-add_subdirectory(simple_solver)
-add_subdirectory(simple_solver_logging)
+option(GINKGO_BUILD_EXTLIB_EXAMPLE "Build the example-6 with deal.II, you need to link the deal.II library." OFF)
+foreach(NUMBER RANGE 1 9)
+  add_subdirectory(example-${NUMBER})
+endforeach()
```

Comment:
Changing everything to `example-<number>` is really a pity, browsing the source code for finding and looking at examples (multiple times eventually) really becomes a nightmare.